README.md

```diff
@@ -49,26 +49,67 @@ Canaveral has support for opening projects in [VSCode](https://code.visualstudio
 
 ### Installation
 
-WARNING: The CLI is still in development, and has yet to be packaged properly. If you want to jump in early, follow the instructions below. Otherwise, stay posted for the next stable release.
+There are **three ways** to install Canaveral: Homebrew, Go get/install, or download the executable.
 
-There are two ways to install Canaveral.
+#### Method 1: Homebrew (MacOS)
 
-#### Go build
+This method might only work for MacOS users, since it requires `brew`.
 
-First, you need to have Go installed. If you don't, follow the instructions [here](https://golang.org/doc/install).
+1. Make sure you have [Homebrew](https://brew.sh/) installed and updated (`brew update`).
 
-Be sure that your GOPATH is configured correctly, so you are able to execute go binaries.
+2. Run:
 
-Then, `go get` Canaveral and install it:
+   ```bash
+   $ brew tap jchengjr77/homebrew-private https://github.com/jchengjr77/homebrew-private.git
+   $ brew install canaveral
+   ```
```

Comment:
This is, and I cannot stress this enough, amazing.

That said, doing `brew uninstall canaveral` does not delete the .canaveral folder. Is there a way to make it to do that? I would assume there is but I'll defer to you.

Reply:
The .canaveral folder is created by us, not homebrew, so we would need an uninstall script to delete that.

For now, I vote we just leave it, since nobody's gonna notice anyway (and it's not an egregiously large folder).